tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -96,7 +102,7 @@ Status RetryingFileSystem::NewRandomAccessFile(
   TF_RETURN_IF_ERROR(CallWithRetries(std::bind(&FileSystem::NewRandomAccessFile,
                                                base_file_system_.get(),
                                                filename, &base_file)));
-  result->reset(new RetryingRandomAccessFile(std::move(base_file)));
+  result->reset(new RetryingRandomAccessFile(std::move(base_file), 0));
```

Comment:
Yes, but zeros should only be passed from the unittest code (as you have implemented in the _test.cc file), whereas the non-test code should still use the actual non-zero values to  have the backoff logic working.